kanidmd/src/lib/actors/v1_read.rs

```diff
@@ -167,6 +171,32 @@ impl QueryServerReadV1 {
         res
     }
 
+    pub async fn handle_live_backup(&self, msg: LiveBackupEvent) {
+        let mut audit = AuditScope::new("live backup", msg.eventid, self.log_level);
+
+        ltrace!(audit, "Begin live-backup event {:?}", msg.eventid);
+
+        let idms_prox_read = self.idms.proxy_read_async().await;
+        lperf_op_segment!(
+            &mut audit,
+            "actors::v1_read::handle<LiveBackupEvent>",
+            || {
+                let res = idms_prox_read
+                    .qs_read
+                    .get_be_txn()
+                    .backup(&mut audit, "/tmp/backup.json");
```

Comment:
This destination needs to be configurable as well as /tmp is not considered a secure location. As well we should consider outputing these backups with a time stamp and rotation policy?

Reply:
Sure. Should I put the config into server.toml then?

Was also thinking of the Rotation... Any good ideas how to do it? Keep last 3 versions?

Somehow like backup-timestamp.json?